pipeline/pipeline.go

```diff
@@ -474,9 +514,34 @@ func (p *Pipeline) expandProcs() {
 	p.procCount.Swap(to)
 }
 
+// DeltaWrapper acts as a wrapper around int64
+// and returns the difference between
+// the new and the old value when a new value is set.
+// This is useful for metrics (see the maintenance function).
+type DeltaWrapper struct {
```

Comment:
I suggest moving `DeltaWrapper` to other file better than declare at 521 line

Reply:
done, thanks